widget/textgrid.go

```diff
@@ -341,17 +378,239 @@ func nextTab(column int, tabWidth int) int {
 }
 
 type textGridRenderer struct {
-	text *TextGrid
+	widget.BaseRenderer
+
+	text   *textGridContent
+	scroll *widget.Scroll
+}
 
-	cols, rows int
+func (t *textGridRenderer) Layout(s fyne.Size) {
+	if t.scroll.Content != nil {
+		t.scroll.Resize(s)
+	}
+	t.text.Resize(s)
```

Comment:
With the suggestion about accessing the objects slice below, this could be:
```suggestion
	t.objects[0].Resize(s)
```